hail/src/main/scala/is/hail/cxx/Compile.scala

```diff
@@ -31,7 +31,7 @@ object Compile {
       Array("Region *" -> "region", "const char *" -> "v"),
       "char *")
 
-    val v = Emit(fb, 1, body)
+    val v = Emit(fb, 1, ir.Subst(body, ir.Env(arg0 -> ir.In(0, arg0Type.virtualType))))
```

Comment:
I think this was wrong before (and the input arguments weren't being used?), since the input argument `Ref` wasn't being substituted with the actual input.

Reply:
Is `cxx.Compile` actually called anywhere?

Reply:
yep--in SparkPipeline.execute and also in assertEvalsTo->nativeExecute.